packages/nimble-components/src/switch/styles.ts

```diff
@@ -107,8 +107,9 @@ export const styles = css`
         border-radius: calc(var(--ni-private-switch-indicator-size) / 2);
         margin: calc(
             calc(
```

Comment:
Really prettier, this is better? 😆

But probably not worth fighting it.

Reply:
Yea, think I'm going to disable prettier on html (something like 1/3rd of all html templates have prettier disable) but for CSS it hasn't been completely unhelpful yet. Though yea, don't particularly like this CSS change but don't feel like fighting it.